### PR TITLE
add comunity tutorials section

### PR DESCRIPTION
**Pull request by atomfrede**
This is related to https://github.com/jhipster/generator-jhipster/issues/11783

I think the tutorial is well written and shows how jhipster is used in reality with both pros and cons. So I have put a new section under "Learn" for community tutorials. What do you think, should we do that?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -597,6 +597,10 @@ <h3>Free Collaborative learning</h3>
 			            <ul>
                             <li>Learn by doing with <a href="https://www.spingular.com" target="_blank" rel="noopener">Spingular</a>. Join a live collaborative project made with Jhipster!: Spingular Chat w/Websockets</li>
                         </ul>
+                        <h3>Community Tutorials/Blogs</h3>
```
Comment (deepu105) [inline index.html:600]: @atomfrede lets merge the section `Online trainings (MOOCs)`, `Free Collaborative learning` and `Community Tutorials/Blogs` together into `Community Tutorials` as they all fall under same kind of stuff and currently having different category is bit confusing. So in the end we will have just
- Books
- Community Tutorials

Comment (atomfrede) [reply index.html:600]: :+1: I thought the same already.

Comment (deepu105) [reply index.html]: done

Comment (atomfrede) [reply index.html]: Thanks!

**Pull request merged**